.github/actions/comment-pr-with-links/comments-with-url-links.js

```diff
@@ -1,51 +1,56 @@
+            owner: context.repo.owner,
+            pull_number: context.issue.number,
+            repo: context.repo.repo,
+        });
+        let result = {};
+        result.updated = prepareLinks({files: FILES.split(' '), siteUrl: SITE_URL, component: COMPONENT_NAME, branch: pr.base.ref});
+        result.deleted = prepareLinks({files: DELETED.split(' '), siteUrl: SITE_URL, component: COMPONENT_NAME, branch: pr.base.ref});
+        return result;
     },
-    createOrUpdateComments: async function ({github,context}){
-        let {LINKS, RENAMED_FILES, HAS_DELETED_FILES} = process.env;
-        const header='## :memo: Check the pages that have been modified\n\n';
-        let body = buildMessage(header,LINKS,HAS_DELETED_FILES === 'true' || RENAMED_FILES != '');
-        const {exists, id} = await githubUtils.isCommentExist({github,context,template: templateCommentId});
+    createOrUpdateComments: async function ({github, context}) {
+        let {LINKS, HAS_DELETED_FILES} = process.env;
+        const header = '## :memo: Check the pages that have been modified \n\n';
+        let links = JSON.parse(LINKS);
+        let body = buildMessage({header, links ,hasWarningMessage : (HAS_DELETED_FILES === 'true')});
+        const {exists, id} = await githubUtils.isCommentExist({github, context, template});
         // Delete oldest comment if another comments exist
-        if (exists && id){
-            await githubUtils.updateComment({github,context,comment_id: id, body});
+        if (exists && id) {
+            await githubUtils.updateComment({github, context, comment_id: id, body});
             return id;
         }
-        const comment = await githubUtils.createComment({github,context, body});
+        const comment = await githubUtils.createComment({github, context, body});
         return comment?.id;
     }
-
 };
 
-function buildMessage(header,links,hasWarningMessage){
+function buildMessage({header, links, hasWarningMessage}) {
     const preface =
         'In order to merge this pull request, you need to check your updates with the following url.\n\n';
 
-    const availableLinks = `### :mag: Page list: \n ${links}\n\n\n\n`;
-
+    const availableLinks = `### :mag: Page list: \n ${links.updated}\n\n\n`;
     let warningAliasMessage = '';
-    if(hasWarningMessage){
-        warningAliasMessage='\n \n ### :warning: At least one page has been deleted in the Pull Request. Make sure to add [aliases](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#use-alias-to-create-redirects)'
+    if (hasWarningMessage) {
+        warningAliasMessage = `### :warning: Alias \n At least one page has been renamed, moved or deleted in the Pull Request. Make sure to add [aliases](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#use-alias-to-create-redirects) and verify if this following link redirect in the right place \n ${links?.deleted}`
```

Comment:
I suggest we introduce redirects in the title as aliases are just the way to create the redirects. We mention 'aliases' in the description

```suggestion
        warningAliasMessage = `### :warning: Check redirects \n At least one page has been renamed, moved or deleted in the Pull Request. Make sure to add [aliases](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/content/CONTRIBUTING.adoc#use-alias-to-create-redirects) and verify that the following links redirect to the right location: \n ${links?.deleted}`
```